src/javascript/8-promises.md

```diff
@@ -77,8 +77,8 @@ the `reason` will be an TypeError object.
 ```js
 const request = fetch("http://foo.bar");
 
-request.catch( function(reason){
-  console.log(reason) /// TypeError: Failed to Fetch
+request.catch( function(error){
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then

it calls the argument `reason` (I've also got it as `reason` in the paragraph above).